test/integration_test/utils/px_operator.go

```diff
@@ -60,3 +63,60 @@ func getPXOperatorImageTag() (string, error) {
 
 	return strings.Split(image, ":")[1], nil
 }
+
+// GetPxOperatorImage return PX Operator image
+func GetPxOperatorImage() (string, error) {
+	deployment, err := appops.Instance().GetDeployment(PortworxOperatorDeploymentName, PxNamespace)
```

Comment:
We could call `GetPxOperatorDeployment(..)` here.

Reply:
Done